lib/src/__tests__/DateRangePicker.test.tsx

```diff
@@ -1,76 +1,22 @@
-// Note that most of use cases are covered in cypress tests e2e/integration/DateRange.spec.ts
 import * as React from 'react';
-import { isWeekend } from 'date-fns';
-import { mount, utilsToUse } from './test-utils';
+import { DesktopDateRangePicker } from '../';
+import { screen } from '@testing-library/react';
+import { utilsToUse, getByMuiTest } from './test-utils';
 import { TextField, TextFieldProps } from '@material-ui/core';
-import { DesktopDateRangePicker } from '../DateRangePicker/DateRangePicker';
+import { createClientRender, fireEvent } from './createClientRender';
 
 const defaultRangeRenderInput = (startProps: TextFieldProps, endProps: TextFieldProps) => (
   <>
-    <TextField {...startProps} />
-    <TextField {...endProps} />
+    <TextField data-mui-test="start-input" {...startProps} />
+    <TextField data-mui-test="end-input" {...endProps} />
```

Comment:
What about using `getAllByRole('textbox')[0]` / `getAllByRole('textbox')[1]` instead adding a `data-mui-test` attribute?

Reply:
data-mui-test here set 1 time, and makes test a little bit easier to read, no? 
But this will work as weell

Reply:
I think that the rule of thumb is that if we can use an accessibility feature, we should. It has more priority. Otherwise, we fallback to a data attribute.

Reply:
Basically, following the same priority rule as [https://testing-playground.com/](https://testing-playground.com/?markup=DwGwhgRgpiAEBmB7ATgXgEQFcDOVkDswBbKdAPgFVcDipgB6caEMgKGAEt8AHTAF1YcAJhhx5CJdKwlRR1GVO7gAxlAAWiEELwYNAd1jYNmLbACSsZWBBwAnokwB%2BKULB8wAWj5RsfYaIVWPz4QWXQofG9kWHtMaLEaSVZrPgwIqJiHePlaKQA3a0wwgCtENXwPIURSINtuMO8ADz4pejZgCH4%2BRHxYGQxsDgBzfExudFg%2BOrDsTAgiDhayQZGxhk6%2Bbvx2oQ482Fd3Lx8-EXQ1GBtECd9bUIw9YT41AC5YAEYABm5GgG5YC7DNR8N5fH7-CBgZQAayGyAc%2BCEb2QUCEvwmbSAA&query=M4YwTgphB2B0DmEAuAhAngFQsJBJAJgBQDkAFhADZUD2xAlEA)